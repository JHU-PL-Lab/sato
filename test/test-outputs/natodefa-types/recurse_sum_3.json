[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [],
          "err_match_val": "true",
          "err_match_expected": "Integer",
          "err_match_actual": "Boolean"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "n + true",
    "err_steps": 2115
  }
]