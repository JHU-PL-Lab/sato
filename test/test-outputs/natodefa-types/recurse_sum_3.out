Type errors for:
- Input sequence : [0]
- Found at expr  : n + true
--------------------
* Value    : true
* Expected : Integer
* Actual   : Boolean
1 errors found. Further control flows may exist.
